tests/ci_build/lint_python.py

```diff
@@ -219,6 +219,8 @@ def main(args: argparse.Namespace) -> None:
                 "tests/ci_build/test_r_package.py",
                 "tests/ci_build/test_utils.py",
                 "tests/ci_build/change_version.py",
+                # Python packaging
+                "python-package/packager/",
```

Comment:
maybe merge this with `python-package/xgboost` and just call mypy with `python-package`?

Reply:
@trivialfis I thought you didn't want to distribute `packager` as part of the Python wheel?

Reply:
Apologies for the ambiguity, I meant running the `mypy` on `xgboost/python-package` on the CI, not distributing `packager`  with xgboost.


https://github.com/dmlc/xgboost/blob/191d0aa5cfac87cfc122ebbe8b14da4177309d13/tests/ci_build/lint_python.py#L201

Reply:
If `packager` is not under directory `xgboost/python-package`, we'd need to specify `packager` as a separate entry. But if we move `packager` to `xgboost/python-package`, then we'd be distributing `packager` with XGBoost.

I prefer the first alternative.

Reply:
My bad, it's sufficient to call `mypy python-package/`